### DISCUSSION TITLE
Request for adding IOMETE platform for implementing Data Mesh architecture

### DISCUSSION THREAD

**Discussion by quantsegu** [datamesh-architecture/datamesh-architecture.com#23] [General]
Request for adding IOMETE as a platform where the "Tech Stack"  can be implemented. I am happy to write the content and get it reviewed with you and make sure it would be provided enough details. 

If this is agreeable i will write the content and share it with you.

IOMETE provides the power of computing at fraction of the cost, we belive in providing peace of mind to data scientists to give them real raw power. With a flat cost model for the platform, we are able to give more ease to the members who are using it.

IOMETE provides an easy to use interface for the developers, analysts and teams.
  

Thanks!
Lakshmi Segu
Developer Advocate 
IOMETE

**Comment by jochenchrist** [comment]
[quantsegu](https://github.com/quantsegu) thanks for your suggestion.

I haven't seen any data mesh built with IOMETE, so for now, I would close this issue as won't fix.
